packages/db/index.js

```diff
@@ -1005,12 +1005,14 @@ export class DBClient {
    * List auth keys of a given user.
    *
    * @param {number} userId
+   * @param {import('./db-client-types').ListKeysOptions} opts
    * @return {Promise<Array<import('./db-client-types').AuthKeyItemOutput>>}
    */
-  async listKeys (userId) {
+  async listKeys (userId, { includeDeleted } = { includeDeleted: false }) {
```

Comment:
Doesn't this prevent us to have the `opts` to be optional?

We want to be able to run `listKeys (1)` and it would error if we don't pass a default opts value in the signature.

Am I missing something?

Reply:
Ahh thats’s a really good point, my bad, I should have had a closer look at the type def